BrowserKit/Sources/ToolbarKit/ToolbarElement.swift

```diff
@@ -17,6 +17,12 @@ public struct ToolbarElement: Equatable {
     /// Mask name of the badge's toolbar element
     let maskImageName: String?
 
+    /// Indiciated whether to use template mode or not for an image
+    let templateModeforImage: Bool
```

Comment:
```suggestion
    /// Indicates whether to use template mode or not for an image
    let templateModeForImage: Bool
```

P.S. I noticed this because of Xcode's spellchecker. Worth enabling if you haven't already!!

<img width="414" height="81" alt="Image" src="https://github.com/user-attachments/assets/742e9ca6-b4f1-40ca-91a3-398f5875295e" />